packages/back-end/src/queryRunners/ExperimentIncrementalRefreshQueryRunner.ts

```diff
@@ -766,6 +756,12 @@ const startExperimentIncrementalRefreshQueries = async (
     params.snapshotType === "standard" && org.settings?.runHealthTrafficQuery;
 
   if (runTrafficQuery) {
+    const dimensionsForTraffic: ExperimentDimensionWithSpecifiedSlices[] =
+      experimentDimensions.flatMap((d) =>
```

Comment:
ooooh I think this is fancy but it threw me off at the start.

wdyt about doing a filter and then a map?